src/tests/domParsers/khanAcademyVideo_tests.ts

```diff
@@ -0,0 +1,102 @@
+/// <reference path="../../../typings/main/ambient/qunit/qunit.d.ts" />
+
+import {VideoUtils} from "../../scripts/domParsers/videoUtils";
```

Comment:
See above comment about splitting up the different domain-specific logics into different class (perhaps implementations of an interface). We can then split the test files up into smaller ones as well. Generally, we want files and their matching test-files to be a 1:1 ratio.